### PR TITLE
Ivan/search name interface

### PR DESCRIPTION
**Pull request by ivanleomk**

### CODE REVIEW

File: client/src/components/ResourceCard.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const ResourceCard = ({terms,name,id}) => {
+    return (
+        <div className = "max-w-sm">
+        <Link to={`/updateResource/${id}`}>
+            
+            <p className = "text-3xl">{name}</p>
+            <div className = "text-xl text-gray-700 mr-6" style = {{maxWidth:"14rem"}}>
+            {terms.map(
+                (item,mapIndex) => <span key = {mapIndex}>{item} {mapIndex < terms.length-1 ? ' | ' : null}</span>
```
Comment (kwannoel) [inline client/src/components/ResourceCard.js:12]: Refactor this to use `divs` and the original interface.

**Comment by kwannoel** [comment]
After comments from @leyliramazanova I've realized that `details` is not necessary.

After search for a name, we should just get names back.

E.g.

Search for `Jon`

| Names |
| - |
| Jonny |
| Jonathan |
| ... |

This should clean up some code from the `details` logic

File: client/src/components/App.js [hunk 2]
```diff
@@ -56,15 +51,22 @@ const App = () => {
...
+                    />
                     
                     <AuthenticatedRoute 
                     exact path="/createResource"
                     component = {CreateResource} /
                     >
-
+                    
+                     
+                    <AuthenticatedRoute
```
Comment (ivanleomk) [inline client/src/components/App.js:65]: This matches any other routes that the user might have keyed in using a wild card character 

see : https://medium.com/@NaveenDA/creating-a-custom-404-notfound-page-with-react-routers-56af9ad67807

File: client/src/components/SearchResourceNameInput.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+const mockData = [
+    {
+        name: "Thomas Frank",
+        personId: 5,
+        contactNumber: "81742341",
+        email: "thomas@thomasfrank.com"
+    },
+    {
+        name: "Thomas Frank",
+        personId: 5,
```
Comment (ivanleomk) [inline client/src/components/SearchResourceNameInput.js:13]: You should replace this id with someone in your local db. That way the link will work nicely when you click into it to the /updateResource/:personId route

Comment (kwannoel) [reply client/src/components/SearchResourceNameInput.js:13]: What's the reason for this mock up?

Comment (kwannoel) [reply client/src/components/SearchResourceNameInput.js:13]: why do we need this mockup

Comment (ivanleomk) [reply client/src/components/SearchResourceNameInput.js:13]: Ahhh, this is to mock the return data if you search for a term. 

It's supposed to be

1. User Keys in a search term
2. we hit a backend API using that search term
3. We get back a list of names and contact details 
4. We render it in the searchResourceNameTable component

I mocked part 2

Comment (kwannoel) [reply client/src/components/SearchResourceNameInput.js:13]: But we don't support this currently

Comment (kwannoel) [reply client/src/components/SearchResourceNameInput.js:13]: All search name is supposed to do is just return us a list of names

Comment (ivanleomk) [reply client/src/components/SearchResourceNameInput.js]: ok. Let me change it then on my end. Ok changed it on my end,

**Pull request merged**